admin/server/usergroups.go

```diff
@@ -446,14 +447,89 @@ func (s *Server) AddProjectMemberUsergroup(ctx context.Context, req *adminv1.Add
 		return nil, err
 	}
 
-	err = s.admin.DB.InsertProjectMemberUsergroup(ctx, usergroup.ID, proj.ID, role.ID)
+	err = s.admin.DB.InsertProjectMemberUsergroup(ctx, usergroup.ID, proj.ID, role.ID, nil)
 	if err != nil {
 		return nil, err
 	}
 
 	return &adminv1.AddProjectMemberUsergroupResponse{}, nil
 }
 
+func (s *Server) AddProjectMemberUsergroupResources(ctx context.Context, req *adminv1.AddProjectMemberUsergroupResourcesRequest) (*adminv1.AddProjectMemberUsergroupResourcesResponse, error) {
```

Comment:
Why does it need a new RPC for this?

I would have expected that you could just pass a `[]ResourceName` list when calling `AddProjectMemberUsergroup`.

Reply:
removed